src/Service/UserAgentInfoHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service;
+
+use App\Entity\UserAgent;
+use Symfony\Component\Finder\Finder;
+
+class UserAgentInfoHandler
```

Comment:
Handler is not the best name for any service. Please read https://refactoring.guru/ if you want to name services the way other developers can understand their purpose